autorest/date/unixtime_test.go

```diff
@@ -1,3 +1,5 @@
+// +build go1.7
```

Comment:
Got it, got it. Disregard this concern, I think it is fair to just be skipping these particular tests in 1.6 if that's the trouble.